matterpy/contrib/rss.py

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env python3
+
+import asyncio
+import feedparser
+
+_conf = None
+_mgr = None
+
+
+def init(manager, conf):
+    global _conf
+    global _mgr
+
+    _conf = conf
+    _mgr = manager
+
+    loop = asyncio.get_event_loop()
+    rss_feeds = [rss_url for rss_url in _conf['rss_feed'].split(',')]
```

Comment:
Suboptimal. The URLs we need may contain commas.

Try to do something like this instead in the config:
```
   feed_foo_url = http://example.org/some_feed
   feed_foo_interval = 30
   feed_bar_url = http://example.com/other_feed
   feed_bar_interval = 600
```
This way, the configuration gets easier for the user, and you don't need to do any shady string ops that may break depending on input.